service/src/test/java/bio/terra/cbas/runsets/inputs/WomToolInputsAndOutputsTest.java

```diff
@@ -0,0 +1,578 @@
+package bio.terra.cbas.runsets.inputs;
```

Comment:
Can this test file be moved to same folder structure as `WomToolInputsAndOutputs.java` under `util/methods`? It seems out of place under `runsets/inputs`